src/store/modules/app.js

```diff
@@ -267,6 +358,9 @@ const mutations = {
     }
     return {};
   },
+  setAppInitialized(state, status) {
+    state.appInitialized = status;
```

Comment:
in the normal case we should never need to toggle this. Maybe just remove status and set it to true

Reply:
no. This is due to this: https://github.com/qmk/qmk_configurator/pull/465/files/f6ff2859823fb65a818063a2b232bf4bf25dabba#diff-872d8e0270fe83b7bae281972a9c2a5dR4

but if we mode the keyboards to props, this state is no more revelant

Reply:
probably not explaining myself well, but when do we ever set `appInitialized = false`?